README.md

```diff
@@ -223,3 +223,22 @@ This will run `eslint --fix` and automatically add changes to the commit. Please
   ]
 }
 ```
+
+### In-place minify the images before adding to the git repo
```

Comment:
Do we need to make the title clearer? 🤔

Reply:
Yes, please re-write the heading

Reply:
@okonet @luftywiranda13 I didn't noticed the review until today. I've reworded the title to the following:
`Minify the images and add files to commit`

Let me know if that's okay. Thanks

Reply:
LGTM!